src/selectedtests/app/controllers/health_controller.py

```diff
@@ -0,0 +1,18 @@
+from flask import jsonify
+from flask_restplus import Api, Resource, fields
+
+
+def add_health_endpoints(api: Api):
```

Comment:
Could you add a docstring to this function?

Reply:
Done